Utilities/CommonDriver.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.IO;
+using TechTalk.SpecFlow;
+
+namespace AdvancedTaskPart2.Utilities
+{
+    public class CommonDriver
+    {
+        public static IWebDriver dr { get; set; }
+
+
+        public static void InitializeWebDriver()
+        {
+            dr = new ChromeDriver();
+            dr.Manage().Window.Maximize();
+            dr.Navigate().GoToUrl("http://localhost:5000/Home");
+        }
+
+        //public static void TearDown()
```

Comment:
please remove comment out code as its bad practice to keep them. The reason is it will confuse other qa's working on the project